package.json

```diff
@@ -21,8 +21,7 @@
     "build": "tsc",
     "example:webpack": "npm run build && cp -rf dist example/webpack/node_modules/angular2-tree-component/ && cd example/webpack && npm start",
     "example:systemjs": "npm run build && cp -rf dist example/systemjs/node_modules/angular2-tree-component/ && cd example/systemjs && npm start",
-    "example:webpack:build": "cd example/webpack && npm run build",
-    "postinstall": "typings install"
```

Comment:
I think the `postinstall` should be changed to `prebuild` so that the development workflow can stay unchanged.  Users of the component won't be bothered by the `prebuild`, but developers can still have typings installed automatically for them.